### ISSUE TITLE
Output settings - Mini Calendar Options?

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#244]
I am continuing to go through the Output Settings screen.

Mini Calendar Options (Not clear what this is) I believe it is a Widget Mini Calendar Options

Target link for mini calendar dates (not clear to understand what this is either).

Actually the whole Mini Calendar Options feels a bit cryptic and needs a better clarity to what it does and is.

![Screenshot 2021-09-24 at 22 27 46](https://user-images.githubusercontent.com/5323259/134735951-399b15a2-d048-4de1-b14d-4213f2b5dc0b.jpg)

**Comment by paaljoachim** [comment]
Grid Options

Mini Calendar Options

List Options

------

Should we instead do something like this?

Calendar Grid Options

Calendar List Options

Widget Mini Calendar Options

**Comment by joedolson** [comment]
Mini is not exclusive to use in the widget; these are controls anywhere that's used, and it can also be used in the shortcode. So it would be misleading to describe it as "widget" options. 

I do wonder sometimes how much anybody uses some of the more obscure options, like linking to an in-page link on the list view. But it's very hard to gather information about that...

**Comment by paaljoachim** [comment]
Let's make it easier to use the Mini Calendar. One way could be to add a mention of the shortcode and the tag which displays the mini calendar.

Below the two fields adding something like:
To use the Mini Calendar as part of the shortcode add the following tag...

**Comment by joedolson** [comment]
I'm not sure "mini" is really the best label for it. Really, it's something like 'Compact View'; that might help, in some ways.

**Comment by paaljoachim** [comment]
I am trying to find some information about the mini calendar but I am drawing a blank.

**Comment by joedolson** [comment]
Not sure what the issue is here; it's just one of the display options. E.g. grid view, list view, mini view, etc. It's not a separate view; just a secondary display option.

**Comment by paaljoachim** [comment]
Where do I find the mini view? How do I use it? 

Ok I noticed it when going to the Shortcode section in the My Calendar page.
![Screenshot 2021-09-27 at 11 18 17](https://user-images.githubusercontent.com/5323259/134880755-e649a248-9a34-4d27-90ab-8f88a54f76a7.jpg)

What I find confusing is that the mini view is kinda hidden away, as one can not using the default switch between List-Grid and Mini. I am working on figuring out why it is there. Examples of how it is used. I understand that one can have a kind of compact Grid view which is the Mini view and that the Mini view is mostly hidden away (almost like an Easter egg in a game. One needs to know where to look to find it.)

Places where I have found a mention of the Mini Calendar Options.
Settings -> Output.
Mobile View to open in Mini Calendar if the user wants.
A Mini Calendar Options area

--

Shortcodes (screen and in meta boxes in My Calendar page)
Format (drop down) option Mini - I selected Mini to see what it would look like on the frontend.

Here is an example of using the theme Twenty Twenty-One and the My Calendar Theme - twentytwenty.css
(Macbook Pro screen 1080p)

![Screenshot 2021-09-27 at 11 34 49](https://user-images.githubusercontent.com/5323259/134883597-e1e356ea-9db1-40f5-80fa-d349bf5e7725.jpg)


---

Using the twentyfourteen.css 

![Screenshot 2021-09-27 at 11 34 16](https://user-images.githubusercontent.com/5323259/134883490-607c7afa-94d1-4332-a7a9-f1bf77e458e0.jpg)

**Comment by joedolson** [comment]
That's...not the mini view. That's the normal grid view in the Week time frame. 

By the way - where's the yellow coming from? Is that a color you set in the CSS variables?

**Comment by joedolson** [comment]
Mini is very much a secondary view. It's really only useful in narrow contexts. The only difference between the mini view and the main grid view is how the pop-up operates. In the main grid view, each event title is listed in the date cell, and can be clicked on separately to trigger a pop-up. In the mini view, you click on the date of the month to view the events occurring on that date.

**Comment by paaljoachim** [comment]
Ahh yes. I just checked the customizer. A month or so ago I added a style to the calendar. That is where the yellow is from.

Ok, so there is not a lot of difference between grid and mini views. Mainly one has the title and the other not. As well as what these look like in the popup card. I will go ahead and close this issue, as I believe I have enough information to create a Calendar mini view doc.
